### ISSUE TITLE
argument residual

### ISSUE THREAD

**Issue by junwon0** [smart-lty/REST#2]
Hello. I want to reproduce the results of your experiments.

I checked the main_results folder and found that there is no residual option in your JSON file. However, the default value in the repository is False. Can you explain the setting for residual?

Thanks

**Comment by smart-lty** [comment]
Hello! Thanks for your interests in our work.

The parameter `residual` correspnds to Line 129 to 159 in `GNN/model/CGNN.py`, which is an attempt to use residual connection to improve the performance. We do not observe significant improvements with residual connection, hence we set the default residual to  False.

**Comment by junwon0** [comment]
Thank you.

I have a problem while reproducing experimental results. My command is: 
`python GNN/train.py --gpu 1 -d fb237_v1 -e fb237_v1 -dim 32 --dropout 0.1 -l 6 --num_epochs 10 --batch_size 32`

Is this the correct command for reproduction of fb237_v1?
